src/dpl/README.md

```diff
@@ -31,15 +39,23 @@ names, or an instance object returned by the SDC `get_cells` command. To
 specify padding for all instances of a common master, use the `-filter`
 "ref_name == <name>" option to `get_cells`.
 
-The `-max_displacement` argument to `detailed_placement` specifies how
-far an instance can be moved when finding a site where it can be placed. The default values are
-`{500 100}` sites. The x/y displacement arguments are in microns.
+```tcl
+set_placement_padding   
+    -global|-masters masters|-instances insts
+    [-right site_count]
+    [-left site_count]
+    [instances]
```

Comment:
Please add a description for the last `[instances]` and clearly differentiate `-instances vs [instances]`.

Reply:
Should we change the name of the `-instances` switch then?

Reply:
@mina1460
Would it be correct to remove `[instances]`? Looking at the code the below looks correct to me.

```suggestion
    -global|-masters masters|-instances insts
    [-right site_count]
    [-left site_count]
```